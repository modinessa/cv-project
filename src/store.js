import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
