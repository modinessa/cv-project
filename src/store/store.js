import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});