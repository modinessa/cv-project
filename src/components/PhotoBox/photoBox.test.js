import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { PhotoBox } from './index';

const TEST_TITLE = 'Title';
const TEST_NAME = 'Name';
const TEST_DESCRIPTION = 'Content bla-bla';
const TEST_AVATAR ='../../assets/images/test.jpeg';

describe('PhotoBox', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<PhotoBox
					name={TEST_NAME}
					title={TEST_TITLE}
					description={TEST_DESCRIPTION}
					avatar={TEST_AVATAR}
				/>
			</Provider>
		)).toMatchSnapshot();
	})
})