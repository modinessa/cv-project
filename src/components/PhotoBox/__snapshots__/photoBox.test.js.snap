// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoBox renders component 1`] = `
<div
  className="app-section_photo-box"
>
  <img
    alt="Name"
    className="app-photo avatar"
    src="../../assets/images/test.jpeg"
  />
  <h1>
    Name
  </h1>
  <h2>
    Title
  </h2>
  <p>
    Content bla-bla
  </p>
</div>
`;
